worldedit-core/src/main/java/com/sk89q/worldedit/function/factory/Deform.java

```diff
@@ -64,6 +70,10 @@ public Deform(Extent destination, Region region, String expression) {
     }
 
     public Deform(Extent destination, Region region, String expression, Mode mode) {
+        this(destination, region, expression, mode, false);
+    }
+
+    public Deform(Extent destination, Region region, String expression, Mode mode, boolean useClipboard) {
```

Comment:
I sort of feel a sourceExtent & sourceRegion would be more useful here as something to pass in rather than useClipboard. Would also allow deform brushes to "bake in" the clipboard so you can have multiple different ones

Reply:
Sorry, I only just saw your comment.
We'd have to offer both, because some people would want the current clipboard, some wouldn't.
Same with the -o flag.
That'd probably a separate PR.

Reply:
baking in is the default for most things and should probably be consistent for ux. (see clipboard brush, `#sel` vs `#dsel` masks, etc)

Reply:
ok I wasn't aware `//brush clipboard` already baked in the clipboard.
I use brushes so rarely :)
guess I'll have to bake that in before merging it then

Reply:
`-o`, on the other hand, is not so simple.
1. Do we read placement type at `//brush` time or at click time?
2. Do we read `pos1` at `//brush` time or at click time?
3. Do we read the clipboard at `//brush` time or at click time?
4. Do we read `player` position at `//brush` time or at click time?

1. Probably at `//brush` time 
2. That'd be the difference between `#sel` and `#dsel`, so we'd probably want to offer both
3. We said we want to bake it in at `//brush` time, but if we offer both in 2., offering both here as well actually saves code
4. I have no idea tbh